Tools/ardupilotwaf/boards.py

```diff
@@ -44,7 +44,7 @@ def configure(self, cfg):
                     keys.sort()
                 val = ['%s=%s' % (vk, val[vk]) for vk in keys]
 
-            if k in cfg.env and isinstance(cfg.env, list):
+            if k in cfg.env and isinstance(cfg.env.k, list):
```

Comment:
Actually it's the same: https://waf.io/apidocs/ConfigSet.html?highlight=configset#waflib.ConfigSet.ConfigSet.__getattr__  But I'll change it.